packages/amplify-e2e-tests/src/utils/retrier.ts

```diff
@@ -0,0 +1,50 @@
+import _ from 'lodash';
```

Comment:
This should go to the e2e-core so we can use this in other places

Reply:
done